pkg/controller/storagecluster/initialization_reconciler.go

```diff
@@ -61,6 +61,23 @@ func (r *ReconcileStorageCluster) ensureStorageClasses(instance *ocsv1.StorageCl
 	return nil
 }
 
+func (r *ReconcileStorageCluster) newOBCStorageClass(initData *ocsv1.StorageCluster) *storagev1.StorageClass {
+	reclaimPolicy := corev1.PersistentVolumeReclaimRetain
+	retSC := &storagev1.StorageClass{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: fmt.Sprintf("%s-ceph-%s-bucket", initData.Name, reclaimPolicy),
+		},
+		Provisioner:   fmt.Sprintf("%s.ceph.rook.io/bucket", initData.Namespace),
+		ReclaimPolicy: &reclaimPolicy,
+		Parameters: map[string]string{
+			"objectStoreName":      generateNameForCephObjectStore(initData),
+			"objectStoreNamespace": initData.Namespace,
+			"region":               "us-east-1",
```

Comment:
I can't imagine we want to hardcode a value here, do we?

Reply:
We do, just like the CSI SCs.